app.rb

```diff
@@ -0,0 +1,76 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'person'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  attr_accessor :books, :people
+
+  def list_books
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def list_people_rentals
+    @people.each_with_index do |person, index|
+      puts "#{index}) Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person(age, name: 'Unknown')
+    person = Person.new(age, name: name)
+    @people.push(person)
+  end
+
+  def create_book(title, author)
+    book = Book.new(title, author)
+    @books.push(book)
+  end
+
+  def create_rental(date, personid, bookid)
+    puts
+    rental = Rental.new(date, @people[personid], @books[bookid])
+    @rentals.push(rental)
+    puts 'Rental created successfully'
+    puts
+  end
+
+  def list_rentals(id)
+    puts
+    @rentals.each do |rental|
+      puts "Date: #{rental.date}, Book \"#{rental.book.title}\" by #{rental.book.author}" if rental.person.id == id
+    end
+    puts
+  end
```

Comment:
- [ OPTIONAL ] I highly recommend while displaying the rental before asking for an id to list all the people to let the user be able to see all the ids before choosing so it would be great if you list the people once the person choose to list the rentals 😊🙏🏻🔥💯✅✅